### PR TITLE
feat: add new command to uninstall extension

### PR DESCRIPTION
**Pull request by gabrielfs7**
Allow uninstall extension via command line

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]
LGTM

File: scripts/uninstallExtension.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ */
+
+use oat\oatbox\service\ServiceManager;
+
+require_once dirname(__FILE__) . '/../includes/raw_start.php';
+
+$parms = $argv;
+array_shift($parms);
+
+if (count($parms) != 1) {
```
Comment (shpran) [inline scripts/uninstallExtension.php:28]:
```suggestion
if (count($parms) !== 1) {
```

**Review by shpran — APPROVED** [review]
Good job 👍🏻

**Review by bartlomiejmarszal — APPROVED** [review]
LGTM

File: scripts/uninstallExtension.php (hunk 1 above)
Comment (gabrielfs7) [reply scripts/uninstallExtension.php]: Done, thanks!

**Pull request merged**